src/esp/assets/ResourceManager.cpp

```diff
@@ -745,6 +747,10 @@ bool ResourceManager::loadPTexMeshData(const AssetInfo& info,
         scene::SceneNode& node = parent->createChild();
         const quatf transform = info.frame.rotationFrameToWorld();
         node.setRotation(Magnum::Quaternion(transform));
```

Comment:
But this was kept?

Reply:
These functions work slightly differently. `loadGeneralMeshData` makes use of `addComponent` to handle mesh hierarchies. `loadPTexMeshData` assumes all meshes are already transformed by all but the global rotation transform (and as written assumes only 1 mesh, `... MeshMetaData(index, index) ...` ). I've added the global transform to the solitary `MeshTransformNode` purely for consistency here, it is likely not to be sued until collision mesh construction.